### PR TITLE
Fix missing checksum

### PR DESCRIPTION
**Pull request by joycetoh8**
This is supposed to fix what happens when a checksum file doesn't currently exist yet to verify the state of twa-manifest.json. This will happen for users updating to the most recent version of Bubblewrap and run `bubblewrap build` without running `bubblewrap update` which will generate the checksum. When `bubblewrap build` doesn't find the checksum file, it will prompt the user whether they would like to generate their project again (which will create checksum) or build without regenerating the project.

### CODE REVIEW

File: packages/cli/src/lib/cmds/build.ts [hunk 1]
```diff
@@ -142,6 +142,25 @@ class Build {
...
   }
 
+  /**
+   * Based on the promptResponse to update the project or not, run an update or print the relevant warning message.
+   */
+  async runUpdate(
+      promptResponse: boolean,
+      manifestFile: string,
+      noUpdateMessage: string): Promise<boolean> {
+    if (promptResponse) {
```
Comment (PEConn) [inline packages/cli/src/lib/cmds/build.ts:152]: nit: do you mind turning this around and making it a guard clause, so:

```
if (!promptResponse) {
  // print message
  return true;
}
```

It just means that the meat of the function is less indented.

File: packages/cli/src/lib/cmds/build.ts [hunk 2]
```diff
@@ -142,6 +142,25 @@ class Build {
         APP_BUNDLE_BUILD_OUTPUT_FILE_NAME, APP_BUNDLE_SIGNED_FILE_NAME);
   }
 
+  /**
+   * Based on the promptResponse to update the project or not, run an update or print the relevant warning message.
```
Comment (PEConn) [inline packages/cli/src/lib/cmds/build.ts:146]: Could you explain what the return value signifies?

File: packages/cli/src/lib/cmds/build.ts [hunk 3]
```diff
@@ -156,20 +175,31 @@ class Build {
...
-        const updated = await updateProject(false, null, this.prompt,
-            this.args.directory, manifestFile);
-        if (!updated) {
-          return false;
-        }
-      } else {
-        this.prompt.printMessage(messages.messageProjectNotUpdated);
+    const targetDirectory = this.args.directory || process.cwd();
+    const checksumFile = path.join(targetDirectory, 'manifest-checksum.txt');
+    let updateResult = true;
```
Comment (PEConn) [inline packages/cli/src/lib/cmds/build.ts:180]: Could this be "updateSuccessful"?

File: packages/cli/src/lib/cmds/build.ts (hunk 1 above)
Comment (joycetoh8) [reply packages/cli/src/lib/cmds/build.ts]: good call out! done :)

File: packages/cli/src/lib/cmds/build.ts (hunk 2 above)
Comment (joycetoh8) [reply packages/cli/src/lib/cmds/build.ts:146]: Added a @return to the method comment

File: packages/cli/src/lib/cmds/build.ts (hunk 3 above)
Comment (joycetoh8) [reply packages/cli/src/lib/cmds/build.ts]: done!

**Review by PEConn — APPROVED** [review]

**Pull request merged**